### PR TITLE
feat(TE): OPTA football widgets

### PR DESCRIPTION
**Pull request by flashcheeks**

### CODE REVIEW

File: packages/article-skeleton/src/article-body/article-body.web.js [hunk 1]
```diff
@@ -178,7 +209,6 @@ const renderers = ({
                   <InArticlePuff
                     sectionColour={theme.sectionColour}
                     forceImageAspectRatio="3:2"
```
Comment (vinhlam) [inline packages/article-skeleton/src/article-body/article-body.web.js:211]: is the santisedhtml required? It doesn't have a default for true does it?

File: packages/ts-components/src/components/opta/football/summary/OptaFootballSummary.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { showcaseConverter } from '@times-components/storybook';
+
+import { ArticleHarness } from '../../../../fixtures/article-harness/ArticleHarness';
+import { OptaFootballSummary } from './OptaFootballSummary';
+
+const showcase = {
```
Comment (vinhlam) [inline packages/ts-components/src/components/opta/football/summary/OptaFootballSummary.stories.tsx:7]: Not for this PR but maybe convert this to a standard storybook so we don't have to rely on `@times-components/storybook`  (using StoryOf works but not the modern format).

File: packages/ts-components/src/components/opta/utils/config.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
+declare global {
```
Comment (vinhlam) [inline packages/ts-components/src/components/opta/utils/config.ts:1]: does this break other things in the window? Should it be in a `*.d.ts` file?

**Review by vinhlam — APPROVED** [review]
Nothing that needs to be done to merge.

**Review by bibicollins1 — APPROVED** [review]
LGTM!

File: packages/article-skeleton/src/article-body/article-body.web.js (hunk 1 above)
Comment (flashcheeks) [reply packages/article-skeleton/src/article-body/article-body.web.js:211]: I'll add it back

File: packages/ts-components/src/components/opta/utils/config.ts (hunk 3 above)
Comment (flashcheeks) [reply packages/ts-components/src/components/opta/utils/config.ts:1]: It does not seem to work in the `*.d.ts` file? Not sure if I need to import/export though 😕

**Pull request merged**